pkg/storage/stores/tsdb/head.go

```diff
@@ -85,6 +87,14 @@ func NewMetrics(r prometheus.Registerer) *Metrics {
 			Name: "loki_tsdb_manager_updates_failed_total",
 			Help: "Total number of tsdb manager update failures (loading/rotating tsdbs in mem)",
 		}),
+		tsdbHeadRotationsTotal: promauto.With(r).NewCounter(prometheus.CounterOpts{
+			Name: "loki_tsdb_head_rotations_total",
+			Help: "Total number of tsdb head rotations",
+		}),
+		tsdbHeadRotationsFailedTotal: promauto.With(r).NewCounter(prometheus.CounterOpts{
+			Name: "loki_tsdb_head_rotations_failed_total",
+			Help: "Total number of tsdb head rotations failed",
+		}),
```

Comment:
you can have just a single metric with `status` label which would have values `success` / `fail`.